src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-form/osd-form.component.ts

```diff
@@ -90,6 +109,12 @@ export class OsdFormComponent extends CdForm implements OnInit {
       }
     };
     this.featureList = _.map(this.features, (o, key) => Object.assign(o, { key: key }));
+    this.deploymentOptionNames = {
+      'cost-capacity': 'Cost/Capacity-optimized ',
+      iops: 'IOPS-optimized',
+      throughput: 'Throughput-optimized'
+    };
```

Comment:
If you use [a string `enum` here](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums), maybe you don't separate names and options classes.

Reply:
Done!